app/views/two_factor_authentication/otp_expired/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title t('titles.otp_expired') %>
+
+<%= render AlertIconComponent.new(icon_name: :error, class: 'display-block margin-bottom-4') %>
+<%= render PageHeadingComponent.new.with_content(t('headings.otp_expired')) %>
+
+  <%= render ButtonComponent.new(
+        action: ->(**tag_options, &block) do
+          link_to(
+            otp_send_path(
+              otp_delivery_selection_form: {
+                otp_delivery_preference: @otp_delivery_preference,
+                resend: true,
+              },
+            ),
+            **tag_options,
+            &block
+          )
+        end,
+        outline: true,
+        icon: :loop,
+        class: 'margin-bottom-neg-1',
+      ).with_content(t('links.two_factor_authentication.send_another_code')) %>
+
+<%= render(
+      'shared/troubleshooting_options',
+      heading_tag: :h2,
+      heading: t('components.troubleshooting_options.default_heading'),
+      options: [
+        {
+          url: login_two_factor_options_url,
```

Comment:
Even though we split out the other troubleshooting option, I think this URL will also need to change depending on whether the user is signing in or creating their account. When clicking this link during account creation, it brings me to "Select your authentication method" screen for login, but I don't have any options because I've not yet set any up.

Also, this screen can be shown when the user adds a new phone number from their account dashboard, where "Choose another authentication method" doesn't really make sense, since they're already signed-in.

![image](https://user-images.githubusercontent.com/1779930/214864258-8d0816f8-c411-4bb7-b5d9-9c479468b1cc.png)

Maybe we should discuss this at standup whether we could incorporate this link as part of the other ticket as well.